src/VuData.h

```diff
@@ -45,8 +47,12 @@ typedef enum VU_UPDATE_STATE
     VU_UPDATE_STATE_NEW
 } VU_UPDATE_STATE;
 
-struct VuEPGEntry 
+class VuEPGEntry 
```

Comment:
IMO it does not make sense to change structs to classes if the only change is to introduce default contructors and to keep all data members public. Please revert those changes.